ansible/roles/rox/tasks/main.yml

```diff
@@ -13,8 +13,9 @@
   shell: composer config --global github-protocols https
 
```

Comment:
ansible's version available in apt-get doesn't support "composer", so the same stuff needs to be done thru "command"